features/apt_messages.feature

```diff
@@ -170,15 +170,10 @@ Feature: APT Messages
         Building dependency tree...
         Reading state information...
         Calculating upgrade...
-
-        CAUTION: Your Ubuntu Pro subscription will expire in 2 days.
-        Renew your subscription at https:\/\/ubuntu.com\/pro to ensure continued security
-        coverage for your applications.
-
         #
-        # News about significant security updates, features and services will
-        # appear here to raise awareness and perhaps tease /r/Linux ;\)
-        # Use 'pro config set apt_news=false' to hide this and future APT news\.
+        # CAUTION: Your Ubuntu Pro subscription will expire in 2 days.
+        # Renew your subscription at https:\/\/ubuntu.com\/pro to ensure continued security
+        # coverage for your applications.
         #
```

Comment:
@lechsandecki 

Today, when a machine has an expiring contract, we insert a message into `apt upgrade` output.

I'm suggesting here that whenever we need to show a contract expired message, that message is displayed in the place of "APT news". The contract expired message would override any "normal" apt news. The contract message would be displayed instead until the contract is renewed or until the user detaches the machine.

I think this cleans up the output and is intuitive. The contract expired messages are very "news-like" and otherwise currently don't fit nicely in the `apt upgrade` output - using the "news" format would fix that IMO

What do you think?